core/platform/search/elastic_search_services_test.py

```diff
@@ -522,38 +453,6 @@ def mock_search(
         self.assertEqual(result, [])
         self.assertIsNone(new_offset)
 
-    def test_blog_post_search_returns_the_right_num_of_docs_even_if_more_exist(
```

Comment:
I think this is a valid scenario to test, you should replace it with an ids_only version of this right?

Reply:
added !